modules/copy-template/go.mod

```diff
@@ -61,11 +61,11 @@ replace (
 
 // Kubernetes
 replace (
-	k8s.io/api => k8s.io/api v0.25.2
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.25.2
-	k8s.io/apimachinery => k8s.io/apimachinery v0.25.2
-	k8s.io/client-go => k8s.io/client-go v0.25.2
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.25.2
+	k8s.io/api => k8s.io/api v0.26.3
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.26.3
+	k8s.io/apimachinery => k8s.io/apimachinery v0.26.3
```

Comment:
This is contradicting the import above?

Reply:
we can't use v0.27, because kubevirt requires v0.26. 
There is a breaking change between v0.26 and v0.27 for kubevirt and until kubevirt onboards newer k8s version we have to stick with v0.26